### PR TITLE
RDM-11245: Add logging for page validation

### PR DESCRIPTION
**Pull request by rajaram-nagarajan**
### JIRA link  ###

https://tools.hmcts.net/jira/browse/RDM-11245

### Change description ###

Added logging for page validation

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: server.ts [hunk 1]
```diff
@@ -51,7 +51,9 @@ const CONFIG = {
   'appInsights_enabled': process.env['APPINSIGHTS_ENABLED'] || 'false',
   'appInsights_roleName': process.env['APPINSIGHTS_ROLE'] || 'ccd-case-management-web',
   'shutter_redirect_url': process.env['SHUTTER_REDIRECT_URL'] || '',
-  'shutter_redirect_wait': parseInt(process.env['SHUTTER_REDIRECT_WAIT'], 10) || 10
+  'shutter_redirect_wait': parseInt(process.env['SHUTTER_REDIRECT_WAIT'], 10) || 10,
+  'logging_level': process.env['LOGGING_LEVEL'] || 'Off',
```
Comment (MSancaktutar) [inline server.ts:55]: `Off` doesn't read like a logging level. Perhaps change var name?

File: server.ts [hunk 2]
```diff
@@ -51,7 +51,9 @@ const CONFIG = {
...
+  'logging_case_field_list': process.env['LOGGING_CASE_FIELD_LIST'] || ''
```
Comment (MSancaktutar) [inline server.ts:56]: would we need this in fact?

Comment (rajaram-nagarajan) [reply server.ts:56]: Yes, we need this to avoid build issues

File: server.ts (hunk 1 above)
Comment (rajaram-nagarajan) [reply server.ts]: Removed 'Off' from the logging level.

File: server.ts (hunk 2 above)
Comment (rajaram-nagarajan) [reply server.ts]: Removed logging

**Review by MSancaktutar — APPROVED** [review]
LGTM

**Pull request merged**